SimPEG/Examples/Mesh_Plot_Cyl.py

```diff
@@ -0,0 +1,66 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from SimPEG import Mesh, Utils, Maps
+
+# Set a nice colormap!
+plt.set_cmap(plt.get_cmap('viridis'))
+
+
+def run(plotIt=True):
+    """
+        Plot Mirrored Cylindrically Symmetric Model
+        ===========================================
+
+        Here, we demonstrate plotting a model on a cylindrically
+        symmetric mesh with the plotting symmetric about x=0.
+    """
+
+    sig_halfspace = 1e-6
+    sig_sphere = 1e0
+    sig_air = 1e-8
+
+    sphere_z = -50.
+    sphere_radius = 30.
+
+    # x-direction
+    cs = 1
+    nc = np.ceil(2.5*(- (sphere_z-sphere_radius))/cs)
+
+    # cell spacings in the x and z directions
+    h = Utils.meshTensor([(cs, nc)])
+
+    # define a mesh
+    mesh = Mesh.CylMesh([h, 1, h], x0 = np.r_[0., 0., -h.sum()/2.])
```

Comment:
right (this was adapted from a notebook where I was shifting things around more... I can update)